CONTRIBUTING.md

```diff
@@ -165,3 +165,27 @@ which tests have failed and then fix them by making more changes. Just use
 `git push pullrequest` to publish your changes. The CI tests will
 run with your updated code. Use `hub ci-status --verbose` to monitor
 their status.
+
+## Flags
+
+V allows you to pass custom flags using `-d my_flag` that can then be checked at compile time (see the documentation about [compile-time if](https://github.com/vlang/v/blob/master/doc/docs.md#compile-time-if))
```

Comment:
These custom flags are most easily used by compiling an instrumented v executable with (for example):
`v -d time_parsing -o vinstrumented cmd/v`, then using the new executable `vinstrumented` instead of v.
I think that should be mentioned too, otherwise people can be confused why just passing them to V, does nothing (because they will affect the compilation of a new v compiler, not other programs).